src/diffusers/models/unets/unet_2d_condition.py

```diff
@@ -762,7 +762,7 @@ def set_default_attn_processor(self):
 
         self.set_attn_processor(processor)
 
-    def set_attention_slice(self, slice_size):
+    def set_attention_slice(self, slice_size: Union[str, int, List[int]] = "auto"):
```

Comment:
this has been done to match the docs @yiyixuxu. if we don't want it to be `auto` by default, i can update the docs accordingly

Reply:
I think it's ok